models/events.model.js

```diff
@@ -22,7 +22,7 @@ var eventsSchema = new mongoose.Schema({
     required: true
   },
   restaurantAddress: {
-    type: String,
+    type: Object,
```

Comment:
Mongoose, unfortunately, doesn't have a type Object. This object will have to be built out.